src/py/flwr_example/quickstart_pytorch/client.py

```diff
@@ -0,0 +1,97 @@
+# Copyright 2020 Adap GmbH. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+from argparse import ArgumentParser
+
+import numpy as np
+import torch
+
+import flwr as fl
+
+from . import mnist
+
+DATA_ROOT = "./data/mnist"
+
+if __name__ == "__main__":
+    # Training settings
+    parser = ArgumentParser(description="PyTorch MNIST Example")
+    parser.add_argument(
+        "--server_address",
+        type=str,
+        default="[::]:8080",
+        help=f"gRPC server address (default: '[::]:8080')",
+    )
+    parser.add_argument(
+        "--cid",
+        type=int,
+        default=0,
```

Comment:
I'd suggest to better remove the default here.

Reply:
Resolved in b1559aeb7d39137b1bdc8f9d5ce5a9dc3a8ca43d